firecloud/fiss.py

```diff
@@ -2805,6 +2805,10 @@ def main(argv=None):
                       'this amount. See Cromwell docs ' +
                       '(https://cromwell.readthedocs.io/en/develop/cromwell_features/RetryWithMoreMemory)' +
                       ' for more information.')
+    subp.add_argument('-p', '--per_workflow_cost_cap', type=float,
```

Comment:
```suggestion
    subp.add_argument('-P', '--per_workflow_cost_cap', type=float,
```
`-p` is reserved for project name

Reply:
done